docs/examples/dimensions.rst

```diff
@@ -558,33 +566,33 @@ dimension. This feature should however be considered experimental.
     productTable = Dimension(
         name='product',
         key='productid',
-        attributes=['name', 'categoryid'],
+        attributes=['name', 'categoryid', 'price'],
         lookupatts=['name'])
 
     categoryTable = Dimension(
         name='category',
         key='categoryid',
-        attributes=['category', 'priceid'],
+        attributes=['category', 'typeid'],
         lookupatts=['category'])
 
-    priceTable = Dimension(
-        name='price',
-        key='priceid',
-        attributes=['price'])
-
-    # A instance of SnowflakedDimension is initialised with the
-    # created dimensions as input, creating a simple interface matching a
-    # single dimension, allowing a Snowflaked dimension to be used in the same
-    # manner as a dimension represented in the database by a Star schema. The
-    # dimensions representing tables are passed in pairs based on their foreign
-    # key relations. Meaning the arguments indicate that the productTable has
-    # a foreign key relation with the categoryTable, and the categoryTable has
-    # a foreign key relation with the priceTable. If a table has multiple
-    # foreign key relations to tables in the Snowflaked dimension, a list must
-    # be passed as the second part of the tuple with a Dimension object for
-    # each table the first argument references through its foreign keys.
+    typeTable = Dimension(
+        name='type',
+        key='typeid',
+        attributes=['type'])
+
+    # A instance of SnowflakedDimension is initialised with the created dimensions
```

Comment:
There was another old typo here: An instance

Reply:
Consider it fixed.